frontend/src/bundles/studio/components/timeline/timecursor/timecursor.tsx

```diff
@@ -178,9 +178,17 @@ const TimeCursor: React.FC<Properties> = ({ playerRef }) => {
             className={styles['timeCursor']}
             role="button"
             tabIndex={0}
+            backgroundColor="brand.secondary.300"
             onMouseDown={handleMouseDown}
         >
-            <Box className={styles['timeCursorArrow']} />
+            <Box
+                className={styles['timeCursorSquare']}
```

Comment:
Do we have css styles for this classes because I can't see them in the project. Seems like you have deleted them when you removed timeline.module.css. I guess we will need to return some styles from that file. But create a separate styles.module.css near timecursor component